Apps.Lexeri/Webhooks/WebhookList.cs

```diff
@@ -0,0 +1,60 @@
+using System.Net;
+using Blackbird.Applications.Sdk.Common.Webhooks;
+using Newtonsoft.Json;
+using Apps.Lexeri.Models.Response;
+using Apps.Lexeri.Webhooks.Handlers;
+
+namespace Apps.Lexeri.Webhooks;
+
+[WebhookList]
+public class WebhookList
+{
+    #region Webhooks
+
+    [Webhook("Term created", typeof(TermCreatedWebhookHandler), Description = "Triggered when a term was created")]
```

Comment:
According to https://docs.blackbird.io/sdk/conventions/#71---event-names we recommend you start all event names with "On ..."